packages/components/psammead-leading-story-promo/src/index.jsx

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+import { node } from 'prop-types';
+import { grid } from '@bbc/psammead-styles/detection';
+import { GEL_GROUP_4_SCREEN_WIDTH_MIN } from '@bbc/gel-foundations/breakpoints';
+import { GEL_SPACING_DBL } from '@bbc/gel-foundations/spacings';
+
+const fourOfSixColumnsMaxWidthScaleable = `66.67%`;
+// (4 / 6) * 100 = 66.6666666667 = 66.67%
+
+const twoOfSixColumnsMaxWidthScaleable = `33.33%`;
+// (2 / 6) * 100 = 0.3333333333 = 33.33%
+
+const LeadingPromoWrapper = styled.div`
+  position: relative;
+
+  @supports (${grid}) {
+    display: grid;
+    grid-template-columns: repeat(6, 1fr);
+    grid-column-gap: ${GEL_SPACING_DBL};
+    grid-column-end: span 6;
```

Comment:
Having `grid-column-end` here makes this difficult to use in other context where the desired span may not be 6.

Reply:
Removed `grid-column-end` definition